support-frontend/assets/helpers/productCatalogue.ts

```diff
@@ -0,0 +1,276 @@
+export const productCatalog = window.guardian.productCatalog;
```

Comment:
Minor nitpick but could we standardise on the spelling of 'catalog' vs 'catalogue'? The filename uses the UK spelling, everywhere else uses the US one. I'd favour the US one as it's what Zuora uses and is used exclusively in support-service-lambdas.